packages/components/src/composite/current/stories/index.story.tsx

```diff
@@ -11,35 +11,31 @@ import { isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	Composite,
-	CompositeGroup,
-	CompositeRow,
-	CompositeItem,
-	useCompositeStore,
-} from '..';
+import * as Composite from '..';
 import { UseCompositeStorePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStorePlaceholder > = {
-	title: 'Components/Composite (V2)',
+	title: 'Components/Composite',
```

Comment:
I opted for keeping `Components/Composite` since the unstable version was using the same title, but happy to discuss alternative options (eg. adding a `/Default` or `/Current` subfolder?)